.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,42 @@
+name: Run integration tests
+
+  pull_request:
+    types: [labeled, synchronize, reopened]
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-22.04
```

Comment:
can we use debian or gardenlinux? it is more similar our prod env.

Reply:
As we want to use public runners whenever possible we are bound to those one here: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources